.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+    "semi": true,
+    "trailingComma": "all"
```

Comment:
When I asked on the call last Tuesday, people didn't seem to have strong opinions about how they wanted to customise these options, but if you want to change any of the rules/settings for Prettier or eslint then please feel free to do so! 😄